lib/rbac/filterer.rb

```diff
@@ -310,8 +310,9 @@ def is_sti?(klass)
     # as done previously.
     def skip_references?(scope, options, attrs, exp_sql, exp_includes)
       return false if scope.singleton_class.included_modules.include?(ActiveRecord::NullRelation)
-      options[:extra_cols].blank? &&
-        (!attrs[:apply_limit_in_sql] && (exp_sql.nil? || exp_includes.nil?))
+      options[:skip_references] ||
```

Comment:
@kbrock well, to say:

> ...or were you thinking only `true`

Is slightly misleading, because you can pass `true` or `false`/`nil` with this currently, just `false`/`nil` will just defer to the other logic.  But I do get what you are getting at.

I think for now, I would like to keep it as is just to keep the fail safe check, just to make sure that anyone that uses this doesn't remove references when they are actually needed in the `where`.  I guess that could lead to it being confusing, but I guess for the most part, I assume that (as you suggested) the `true` will be the only value that is used.

I guess this is a round-about way of saying I don't have a real strong feeling one, so what do you think?